### ISSUE TITLE
configuration of the ablation experiment

### ISSUE THREAD

**Issue by AliceCrystal** [arunos728/MotionSqueeze#5]
Thanks for your great job! Could you share the configuration of TSM ResNet-18, including training eopchs, batch_size, initial learning rate and learning rate decay ? For the Something-Something V1&V2.  Would the hypermeter configuration change if you add your Innovation Point module?

**Comment by AliceCrystal** [comment]
> Thanks for your great job! Could you share the configuration of TSM ResNet-18, including training eopchs, batch_size, initial learning rate and learning rate decay ? For the Something-Something V1&V2. Would the hypermeter configuration change if you add your Innovation Point module?

**Comment by arunos728** [comment]
The hyper-parameter configuration of TSM/MS ResNet-18 is at (scripts/train_TSM_Something_v1.sh ) in our github. For both Something-V1 & V2, we use 40 epochs, 48 batch size (4 gpus), and 0.01 initial learning rate. The learning rate is decayed by 1/10 after 20 and 30 epochs.

**Comment by AliceCrystal** [comment]
got it！thanks again！
